pkg/util/template/template.go

```diff
@@ -11,7 +12,11 @@ var templateValidator = NewValidator(
 	MaxDepth(15),
 )
 
-var templateFuncMap = map[string]interface{}{
-	messageformat.TemplateRuntimeFuncName: messageformat.TemplateRuntimeFunc,
-	"makemap":                             MakeMap,
+func MakeTemplateFuncMap() map[string]interface{} {
+	var templateFuncMap = sprig.FuncMap()
+	templateFuncMap[messageformat.TemplateRuntimeFuncName] = messageformat.TemplateRuntimeFunc
+	templateFuncMap["makemap"] = MakeMap
```

Comment:
Can you help replacing our custom `makemap` function with sprig's `dict` function in templates?

Reply:
Fixed